apps/src/templates/tables/QuickActionsCell.jsx

```diff
@@ -136,6 +136,7 @@ export default class QuickActionsCell extends Component {
           style={iconStyle}
           onClick={this.state.canOpen ? this.open : undefined}
           className="ui-test-section-dropdown"
+          className="ui-projects-table-dropdown"
```

Comment:
Multiples classNames make the linter grumpy since they're considered duplicate prop declarations.  Is the best practice here to pick one generic name to use across UI tests? Or to set className based on where the component is used via another prop? Or something else?

Reply:
I think you can use multiple classname in a single string separated by spaces?

https://github.com/code-dot-org/code-dot-org/blob/8baef740ebddc680448e51465263328bb46bddb6/apps/src/templates/instructions/TeacherFeedback.jsx#L134